review_wof/model/gerrit_patches.py

```diff
@@ -18,14 +16,18 @@
 
 
 class GerritPatches(GeneratorDSC):
-    def __init__(self, since=None, until=None):
+    def __init__(self, host='gerrit.ovirt.org', since=None, until=None):
```

Comment:
I actually do not think GerritPatches should be a single-server - based class, instead I think we need to expand it internally to be able to collect information from multiple servers.

Reply:
Agree. Then, do we want to remove the default value?